src/scala/org/pantsbuild/zinc/scalautil/ScalaUtils.scala

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2018 Pants project contributors (see CONTRIBUTORS.md).
+ * Licensed under the Apache License, Version 2.0 (see LICENSE).
+ */
+
+package org.pantsbuild.zinc.scalautil
+
+import java.io.File
+import sbt.io.Path
+import java.net.URLClassLoader
+import sbt.internal.inc.ScalaInstance
+import xsbti.compile.{
+  ScalaInstance => XScalaInstance
+}
+import org.pantsbuild.zinc.util.Util
+
+object ScalaUtils {
```

Comment:
I could buy that this should live in `CompilerUtils`, or I can buy that it is worth being a standalone thing. Any strong feelings either way?

Reply:
That would mean that the `:bootstrapper` would depend on `:compiler` just for this common function, and I don't think that is a strong enough reason since the function is not intrinsic to the domain of the compiler.